dev/Installer/src/install.cpp

```diff
@@ -0,0 +1,172 @@
+#include "pch.h"
+#include "packages.h"
+#include "install.h"
+
+using namespace winrt;
+using namespace Windows::ApplicationModel;
+using namespace Windows::Foundation;
+using namespace Windows::Management::Deployment;
+using namespace Windows::System;
+
+winrt::hresult add_package(Uri packageUri)
```

Comment:
NIT: THese methods use _'s to separate words.  We should come to a conclusion on what format our method names should be.

Reply:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#nl8-use-a-consistent-naming-style

@DefaultRyan or @kennykerr probably have casing opinions.  My guess is they'd want `snake_case` but I'm an old fogey who's partial to `PascalCasing`.

Reply:
@jonwis I'm in your camp as well, to align with .NET. Are there coding guidelines coming soon?

Reply:
I don't have an opinion. For reference, The C++/WinRT library distinguishes between projected types and library types by using snake_case for the latter, but other projects/libraries are free to do otherwise.

Reply:
m_ PascalCase for member variables
CONSTANTS_THAT_HAVE_SPACES
PascalCase for methods and classes
camelCase for local variables

Reply:
@dhoehna yes, except for the first one - `m_camelCase` is a scoped variable, so it gets `m_` and `camelCase`.

Reply:
My intent here was to try to comply with the https://github.com/isocpp/CppCoreGuidelines by using PascalCase for types and preferring snake_case everywhere else (per [NL.10: Prefer underscore_style names](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#nl10-prefer-underscore_style-names)

However, NL 10 also states that it is only if I 'have a choice' - since this was a new project, I assumed there was a choice here and preferred underscore except for Type names, but from feedback it appears what is desired is Windows-style guidelines which @dhoehna describes above.

I'm perfectly OK with this. I personally find snake_case difficult to read and awkward, and I prefer the Windows-style. Since MRT Core has uses Windows style and has gone through review I will assume this is an acceptable naming convention and convert all naming in this project over to Windows-style and ignore NL 10 in the CppCoreGuidelines.

I've created issue #452 to get some clarity on what is preferred for this project with regards to NL 10.

Reply:
I aligned all around the standard Windows style in the latest push.